grep-9999-grnet_tcpudp.md

```diff
@@ -0,0 +1,48 @@
+# GREP [XXXX] -- Upstreaming gr-grnets TCP and UDP Source/Sink Blocks
+
+- Original Author: Michael Piscopo <ghostop14@gmail.com>
+- Champion: Michael Piscopo <ghostop14@gmail.com>
+- Status: Draft
+
+History:
+- 07-Jan-2020: Initial Draft
+
+## Abstract
+
+While ZMQ provides a very efficient way to move data when the developer owns both sides 
+of the link, legacy systems and systems outside a developer's control still employ TCP 
+and UDP data streams.  The gr-grnet TCP and UDP blocks have been completely refactored 
+over the past year and written entirely in boost for the best performance. UDP source and 
```

Comment:
I suppose it's mostly `asio` and probably can even be (soon) replaced by standalone `asio` which is the canonical upstream now.